src/sidebar/components/Excerpt.tsx

```diff
@@ -95,19 +113,19 @@ function Excerpt({
   onToggleCollapsed = noop,
   overflowThreshold = 0,
   settings = {},
-}) {
+}: ExcerptProps) {
   const [collapsedByInlineControls, setCollapsedByInlineControls] =
     useState(true);
 
-  const contentElement = /** @type {{ current: HTMLDivElement }} */ (useRef());
+  const contentElement = useRef<HTMLDivElement>();
 
   // Measured height of `contentElement` in pixels
   const [contentHeight, setContentHeight] = useState(0);
 
   // Update the measured height of the content container after initial render,
   // and when the size of the content element changes.
   const updateContentHeight = useCallback(() => {
-    const newContentHeight = contentElement.current.clientHeight;
+    const newContentHeight = contentElement.current?.clientHeight ?? 0;
```

Comment:
I'm not 100% sure if falling back to `0` is the right thing to do. It should never happen, as `current` will usually not be `undefined`, but might be worth reviewing.

Reply:
In these cases, when we know `ref.current` should be set, we've been using `!`